apps/admin_app/lib/admin_app_web/mailer/order_export_mail.ex

```diff
@@ -0,0 +1,20 @@
+defmodule AdminAppWeb.OrderExportMail do
```

Comment:
This module name should be generic, it will also be used for product export feature.